src/utils/setup/on-create-page.js

```diff
@@ -42,7 +42,9 @@ const findArticlesFromSlugs = (allArticles, articleSlugs, maxSize) => {
     // If maxSize is undefined, this will return a shallow copy of articleSlugs
     articleSlugs.slice(0, maxSize).forEach((featuredSlug, i) => {
         const newArticle = allArticles.find(
-            x => x.query_fields.slug === featuredSlug
+            x =>
```

Comment:
The tests caught an interesting case where the featured article slug was a substring of another article slug so i was not getting the correct response. Checking strict equality here is better than `indexOf` for that reason.